packages/jest-snapshot/src/utils.ts

```diff
@@ -19,7 +19,7 @@ import type {SnapshotData} from './types';
 
 export const SNAPSHOT_VERSION = '1';
 const SNAPSHOT_VERSION_REGEXP = /^\/\/ Jest Snapshot v(.+),/;
-export const SNAPSHOT_GUIDE_LINK = 'https://goo.gl/fbAQLP';
+export const SNAPSHOT_GUIDE_LINK = 'https://jestjs.io/docs/snapshot-testing';
```

Comment:
Oh.. That’s a good point.

I would say it should stay as is. The number is more like a version of `.snap` syntax. If there would be some change in the syntax of `.snap` file, this would indicate that v1 reader will not be able to interpret v2 `.snap` correctly. Which is not the case here. (Sorry, about this clumsy explanation.)